### PR TITLE
feat: add interface docs

### PR DESCRIPTION
**Pull request by gangb-tech**

### CODE REVIEW

File: src/main/java/com/featureprobe/sdk/server/FPUser.java [hunk 1]
```diff
@@ -3,41 +3,75 @@
...
 
     private Map<String, String> attrs = new HashMap<>();
 
-
-    public FPUser with(String key, String value) {
-        attrs.put(key, value);
-        return this;
+    /**
+     * Creates a new FPUser
+     * @param key unique identifier
```
Comment (FengShang) [inline src/main/java/com/featureprobe/sdk/server/FPUser.java]: user unique id for percentage rollout.

File: src/main/java/com/featureprobe/sdk/server/FPUser.java [hunk 2]
```diff
@@ -3,41 +3,75 @@
...
+    public FPUser with(String name, String value) {
+        attrs.put(name, value);
+        return this;
     }
 
-
-    public FPUser(String key) {
-        this.key = key;
+    /**
+     * Check property exists
```
Comment (dengfeige) [inline src/main/java/com/featureprobe/sdk/server/FPUser.java]: Check attribute exists ?

File: src/main/java/com/featureprobe/sdk/server/FeatureProbe.java [hunk 3]
```diff
@@ -38,10 +41,19 @@ private FeatureProbe(DataRepository dataRepository) {
         eventProcessor = config.eventProcessorFactory.createEventProcessor(context);
     }
 
+    /**
+     * Creates a new client instance that connects to FeatureProbe with the default configuration.
+     * @param sdkKey for your FeatureProbe environment
```
Comment (dengfeige) [inline src/main/java/com/featureprobe/sdk/server/FeatureProbe.java]: sdkKey -> serverSDKKey ?

**Comment by FengShang** [comment]
看一下Codacy 的静态检查建议

**Pull request merged**